product/infrastructure/fake/helper.go

```diff
@@ -60,3 +64,36 @@ func unmarshalJSONProduct(productRaw []byte) (domain.BasicProduct, error) {
 
 	return *simpleProduct, nil
 }
+
+//go:embed testdata
```

Comment:
seems, that we just need `testdata/categoryFacetItems.json`, so we could embed that file directly